### PR TITLE
Readme up

### PR DESCRIPTION
**Pull request by jperez999**
This PR updates the Readme for systems. Adds a few new sections and updates some of the ones already in place.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,82 @@ You can install Merlin Systems with `pip`:
 pip install merlin-systems
 ```
 
+### Installing Merlin Systems from Source
```
Comment (karlhigley) [inline README.md:23]: Let's move this section to a CONTRIBUTING.md file. Maybe we can steal one from another Merlin repo?

File: README.md [hunk 2]
```diff
@@ -20,6 +20,82 @@ You can install Merlin Systems with `pip`:
...
+
+Merlin Systems can be installed from source by cloning the GitHub repository and running `setup.py`
+
+```bash
+git clone https://github.com/NVIDIA-Merlin/systems.git
+cd systems && python setup.py develop
+```
+
+### Running Merlin Systems from Docker
```
Comment (karlhigley) [inline README.md:32]: I think we might want to keep the table of which libraries are installed on which containers at the Merlin level

File: README.md [hunk 3]
```diff
@@ -20,6 +20,82 @@ You can install Merlin Systems with `pip`:
...
+Merlin Systems is installed on multiple Docker containers, which are available in the [NVIDIA Merlin container repository](https://catalog.ngc.nvidia.com/containers?filters=&orderBy=dateModifiedDESC&query=merlin):
+
+| Container Name             | Container Location | Functionality |
+| -------------------------- | ------------------ | ------------- |
+| merlin-inference           | https://catalog.ngc.nvidia.com/orgs/nvidia/teams/merlin/containers/merlin-inference           | Merlin frameworks and Triton Inference Server |
+| merlin-tensorflow-inference            | https://catalog.ngc.nvidia.com/orgs/nvidia/teams/merlin/containers/merlin-tensorflow-inference            | Merlin frameworks selected for only Tensorflow support and Triton Inference Server                    |
+
+If you want to add support for GPU-accelerated workflows, you will first need to install the [NVIDIA Container Toolkit](https://github.com/NVIDIA/nvidia-docker) to provide GPU support for Docker. You can use the NGC links referenced in the table above to obtain more information about how to launch and run these containers.
+
+## Getting Started with Merlin Systems
```
Comment (karlhigley) [inline README.md:43]: How similar is this to the example notebook? Could we maybe condense it down to a quick example that shows what it looks like to build and export a simple ensemble and refer people to the notebook for details?

**Review by karlhigley — APPROVED** [review]
Looks like good content got added here! Based on conversations with @mikemckiernan, we might want to move some of this into other files/repos—I'm going to go ahead and approve the PR assuming we can make further edits in additional PRs.

File: README.md [hunk 4]
```diff
@@ -1,13 +1,100 @@
...
+)
+
+retrieval = (
+    user_features
+    >> PredictTensorflow(retrieval_model_path)
+    >> QueryFaiss(faiss_index_path, topk=100)
+)
+
+# Filter out candidate items that have already interacted with
+# in the current session and fetch item features for the rest
```
Comment (mikemckiernan) [inline README.md:72]: sugg: s/the rest/the remaining candidate items/

File: README.md [hunk 5]
```diff
@@ -1,13 +1,100 @@
...
+Once you have successfully exported your ensemble you can use the created directory to run an instance of tritonserver hosting your ensemble.
+```shell
+tritonserver --model-repository=/export_path/
+```
+
+See the [Getting Started with Merlin Systems](./examples/Getting_Started/Getting-started-with-Merlin-Systems.ipynb) example notebook for details. The notebook deploys this ensemble and demonstrates sending requests to Triton Inference Server.
+
+## Building a Four-Stage Recommender Pipeline
+
+Merlin Systems can also build more complex serving pipelines that integrate multiple models and external tools (like feature stores and nearest neighbor search):
```
Comment (mikemckiernan) [inline README.md:46]: Maybe "...and external tools such as feature stores and nearest neighbor search. The following code sample shows the key classes and methods for integrating with a feature store:"

File: README.md [hunk 6]
```diff
@@ -1,13 +1,100 @@
...
+)
+
+# Create and export the ensemble
+ensemble = Ensemble(ordering, request_schema)
+ensemble.export("./ensemble")
+```
+
 ## Installation
 
 Merlin Systems requires Triton Inference Server and Tensorflow. The simplest setup is to use the [Merlin Tensorflow Inference Docker container](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/merlin/containers/merlin-tensorflow-inference), which has both pre-installed.
```
Comment (mikemckiernan) [inline README.md:100]: nit: s/Tensorflow/TensorFlow/g

File: README.md (hunk 1 above)
Comment (mikemckiernan) [reply README.md:110]: +1 that this is better in CONTRIBUTING.md along with the suggestion to install pre-commit and that docstrings are numpy flavor.  I need more hours in my days.

**Review by mikemckiernan — APPROVED** [review]
This looks good. Thank you immensely for putting it together.

**Pull request merged**